src/scripts/app.ts

```diff
@@ -22,24 +27,15 @@ const playButton = document.querySelector(
 const restartButton = document.querySelector(
 	'.js-restart-game'
 ) as HTMLButtonElement;
-// Hidden class
-const hiddenClass = 'hidden';
 
 const imgItemsArray: Array<HTMLImageElement> = [];
 
 const fetchArray: Basic[] = [];
 
-// attempts
-let ATTEMPT_COUNTER = 0;
-const MAX_ATTEMPTS = 2;
-
-let PLAYER_SCORE = 0;
-let COMPUTER_SCORE = 0;
-
 const validateAndCloseModal = () => {
 	if (nameInput.value.trim()) {
-		modal.classList.add(hiddenClass);
-		gameSection.classList.remove(hiddenClass);
+		modal.classList.add('hidden');
+		gameSection.classList.remove('hidden');
```

Comment:
Były zmienne dobrze, czemu powrót do stringa? Ten string powinien być np HIDDEN_CLASS nazwany, bo to absolutna stała.